src/constants.py

```diff
@@ -15,6 +15,7 @@
     '2016-17': ['20161025', '20170412'],
     '2017-18': ['20171017', '20180411'],
     '2018-19': ['20181016', '20190410'],
+    '2019-20': ['20101022', '20200410']
```

Comment:
should be ['20191022', '20200410']?

Reply:
Oh! Of course. I think i wrote it accidentally.